core/templates/pages/collection-player-page/collection-local-nav/collection-local-nav.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <ul class="nav navbar-nav oppia-navbar-nav float-right oppia-navbar-pencil-icon">
+    <li *ngIf="canEdit" class="nav-item">
+      <a [href]="'/collection_editor/create/' + collectionId" ngbTooltip="Edit" placement="bottom" target="_blank" rel="noopener" class="nav-link">
```

Comment:
instead of caluclating this value here, should we add all the logics in the .ts file? (either have a collectionEditorPgaeURL variable or getCollectionEditPageUrl function)

Reply:
After reviewing other files I realized that we had this format earlier, marking this comment as _optional_, feel free to resolve this comment if it seems simple for you! :)

Reply:
I'm not getting this one can you please give some example from the codebase?